libs/wire-api/src/Wire/API/ErrorDescription.hs

```diff
@@ -370,3 +370,9 @@ type BroadcastLimitExceeded =
     400
     "too-many-users-to-broadcast"
     "Too many users to fan out the broadcast event to."
+
+type SecondFactorAuthDisabled =
```

Comment:
```suggestion
type 2FPasswordChallengeDisabled =
```

...  or whatever we agree on in https://wearezeta.atlassian.net/browse/SQSERVICES-1159

(please also adjust label and message below, servant route, ....)

Reply:
AFAIK, identifiers can not start with a numeric character.

Reply:
What about `SndFPasswordChallengeDisabled` then?

Reply:
`PasswordChallengePlusCode`?  i have no good idea.  your suggestion is good enough.